jboss-eap-logging-jdk11/added/module.xml

```diff
@@ -0,0 +1,16 @@
+<module name="org.jboss.logmanager" xmlns="urn:jboss:module:1.8">
+    <resources>
+        <resource-root path="jboss-logmanager-2.1.7.Final.jar"/>
```

Comment:
This shouldn't be needed once 7.2.1 is the base zip as it will have this version.